src/AppContainer.dev.js

```diff
@@ -69,11 +69,23 @@ class AppContainer extends Component {
     if (error) {
       return <this.props.errorReporter error={error} />;
     } else {
-      return <this.props.component {...this.props.props} />;
+      return React.Children.only(this.props.children);
     }
   }
 }
 
+AppContainer.propTypes = {
+  children: function (props, propName, componentName, location, propFullName) {
+    if (typeof props[propName].type !== 'function') {
+      return new Error(`Invalid prop ${propFullName} supplied to ${componentName}. Expected a single React element with your app’s root component, e.g. <App />.`);
+    }
+
+    if (React.Children.count(props[propName]) > 1) {
+      return new Error(`Invalid prop ${propFullName} supplied to ${componentName}. Expected a single React Component!`);
```

Comment:
Error message needs to be updated here too. It’s fine if it’s exactly the same one.